az-digital/az_quickstart
# Fixes #756 Embedded Entity Migration

This provides a process plugin for embedded entities that parses HTML and translates embedded entity IDs according to migration lookups.

## Description
Fixes #756 

## Related Issue
#756 

## How Has This Been Tested?
- Create text with embedded entities in a source d7 system
- Verify if embedded entities are migrated into d9 when migrations are run

## Types of changes


## Checklist:
